src/styles/blocks/catalog.scss

```
@@ -17,9 +17,16 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: color $time, transform $time;
```

```suggestion
    transition-property: color, transform;
    transition-duration: $time;
```
Just for your information, it can also be written as follow.
In your case it's okay, but if you have a lot of properties, then it's better to use the following variant